images/vm-route-forge/Taskfile.yaml

```diff
@@ -2,7 +2,34 @@
 
 version: "3"
 
+vars:
+  DLV_IMAGE_ENV: "DLV_IMAGE"
+
 tasks:
   lint:
     cmds:
       - golangci-lint run --sort-results
+
+  gen:
+    desc: "Generate"
+    cmd: go generate ./...
+
+  dlv:build:
+    desc: "Build image vm-route-forge with dlv"
+    preconditions:
+      - sh: if [ -z $DLV_IMAGE ]; then exit 1 ; fi
+        msg: "export env {{ .DLV_IMAGE_ENV }}"
+    cmd: cd ../../ && docker build -f ./images/vm-route-forge/dlv.Dockerfile -t "${DLV_IMAGE}" .
+
+  dlv:push:
+    desc: "Push image vm-route-forge with dlv"
+    preconditions:
+      - sh: if [ -z $DLV_IMAGE ]; then exit 1 ; fi
+        msg: "export env {{ .DLV_IMAGE_ENV }}"
+    cmd: cd ../../ && docker build -f ./images/vm-route-forge/dlv.Dockerfile -t "${DLV_IMAGE}" .
```

Comment:
1. These tasks are same: cmd is docker build in both cases.
2. Why define DLV_IMAGE_ENV if `$DLV_IMAGE` is used ins script?
3. `msg: "export env {{ .DLV_IMAGE_ENV }}"` -> `msg: "Set DLV_IMAGE to dlv image name, e.g. <add example image name>`
4. Why there is no default dlv image?